src/index.js

```diff
@@ -6,3 +6,33 @@ import "./index.scss";
 
 // install snippets styles
 import "highlight.js/styles/default.css";
+
+window.onload = () => {
```

Comment:
I'd prefer using DOM Events than `window.onload`. Can we replace this with a `DOMContentLoaded` event handler? I'm not sure how that interacts with the js bundle being an async module though.

```suggestion
document.addEventListener("DOMContentLoaded", () => {
```

Reply:
If this file gets compiled to `main.bundle.js`, replace `async` with `defer` in the HTML and you don't even need an event handler, just execute it immediately.

Reply:
changed to defer.